src/hexo/themes/sonarwhal/source/core/css/scan-results.css

```diff
@@ -16,6 +16,32 @@
     padding: 1rem 2rem;
 }
 
+.scan-overview__header p:first-child {
+    margin-right: 5rem;
+    margin-bottom: 1rem;
+}
+
+.scan-overview__status {
+    margin: 0;
+    text-transform: capitalize;
+}
+
+#status-analyzing {
+    animation: fade 3s linear infinite;
+}
+
+@-webkit-keyframes fade {
```

Comment:
I don't think we need the prefixed version. Chrome v43 added the unprefixed version and we are now in v62